utils/chat.js

```diff
@@ -14,7 +14,7 @@ const determineMessageType = (messageType, attachments) => {
  * @param {import('stream-chat').MessageResponse} message
  */
 const generateReplyDataFromMessage = (message, userData, isAnonimous = true) => {
-  const anonymousName = `Anonymous ${userData?.anon_user_info_emoji_name}`;
+  const anonymousName = `${userData?.anon_user_info_color_name} ${userData?.anon_user_info_emoji_name}`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The variable `isAnonimous` is not used in the function. If it's intended to control whether to use the anonymous name or not, you should add a condition to check its value.
```diff
- const anonymousName = `${userData?.anon_user_info_color_name} ${userData?.anon_user_info_emoji_name}`;
+ const anonymousName = isAnonimous ? `${userData?.anon_user_info_color_name} ${userData?.anon_user_info_emoji_name}` : userData?.username;
```
This change will ensure that the username is used when `isAnonimous` is false, and the anonymous name is used when `isAnonimous` is true.


<!-- This is an auto-generated comment by OSS CodeRabbit -->